### ISSUE TITLE
Any solution for using HPSv2 evaluation model offline?

### ISSUE THREAD

**Issue by love-yiyi** [tgxs002/HPSv2#16]
Many GPU servers may not have direct Internet connection, is there a simple way to work around?

**Comment by tgxs002** [comment]
Good point, will update a tutorial on that when time is available. You can try running it on your local machine, and then upload the cached files (~/.cache/hpsv2/ by default) to the corresponding place on your remote machine via other protocols, like FTP.

**Comment by Vei-son** [comment]
I am still confused about how to use HPSv2 evaluation model offline. I have download the HPSv2 checkpoint in my local machine, and upload this checkpoint file to the appropriate place on my GPU server. And I tried many ways to place the checkpoint, but it still needed to download the checkpoint from Huggingface. So what should I do if I can't settle the Internet connection problem? Thanks.

**Comment by NormXU** [comment]
@Vei-son I agree. After reviewing the code, I found it's set up to check the model configuration online. Maybe we can change it to let users use the evaluation model offline? @tgxs002

**Comment by RollingWang** [comment]
@tgxs002 Thanks for your great job. I also want to utilize hpsv2 to evaluate my various T2I models offline, because all my servers are offline, but it's challenging and failed again and again, can you give some ideas?

**Comment by yutongtonghu** [comment]
> I am still confused about how to use HPSv2 evaluation model offline. I have download the HPSv2 checkpoint in my local machine, and upload this checkpoint file to the appropriate place on my GPU server. And I tried many ways to place the checkpoint, but it still needed to download the checkpoint from Huggingface. So what should I do if I can't settle the Internet connection problem? Thanks.

hi, do you have solved this problem.
